content/events/2022-11-17-community-call/index.md

```diff
@@ -16,7 +16,7 @@ subsites: [all]
 
 We will have some community updates followed by a presentation on [**The new GTN course builder**]() by **Helena Rasche & Saskia Hiltemann**. If you'd like to discuss particular topics, please add them to the [agenda](https://docs.google.com/document/d/1PYUnO_td7f-l0fxRYDn8D6ER7W6eY-E9mUx8ErT8kJ4/edit?usp=sharing) or bring them to the call!
 
-**Please [join us via zoom](https://zoom.us/j/93232917829?pwd=NThHTkdDYmFTRExuV0VvaE1vRmZ1UT09) at 3 pm UTC** ([see in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Galaxy+community+call&iso=20221027T15)) for an open discussion.
+**Please [join us via zoom](https://zoom.us/j/93232917829?pwd=NThHTkdDYmFTRExuV0VvaE1vRmZ1UT09) at 4 pm UTC** ([see in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Galaxy+community+call&iso=20221027T15)) for an open discussion.
```

Comment:
Is this right?  This was at 11 EDT today, which should have been 3 UTC according to the timeanddate link.

Reply:
well I would say, its wrong in the minutes as well...
You changed from EDT to EST beginning of November, didn't you?
EST is afaik 5 hours behind UTC